provision/acc_provision/templates/provision-config.yaml

```diff
@@ -59,6 +59,11 @@ net_config:
                                    # CNI should not wait for datapath to be
                                    # ready before returning success for pod
                                    # interface creation
+ #duration_wait_for_network: 210  # Duration in seconds that ACI should wait
```

Comment:
The duration_wait_for_network value must be a string, since that is what is expected in aci_deployment.yaml.